src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        long start = System.nanoTime();
+        final SimpleDateFormat formatter = new SimpleDateFormat("dd.MM.y");
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/exceptions/salary-info): If you create a [formatter](https://docs.oracle.com/javase/tutorial/datetime/iso/format.html), make it a constant field. https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names